csharp/ql/lib/semmle/code/cil/Method.qll

```diff
@@ -89,7 +89,7 @@ class Method extends DotNet::Callable, Element, Member, TypeContainer, DataFlowN
 
   override Location getLocation() { result = Element.super.getLocation() }
 
-  override Location getALocation() { cil_method_location(this.getUnboundDeclaration(), result) }
+  override Location getALocation() { cil_method_location(this.getUnboundMethod+(), result) }
```

Comment:
Why do we need to transitively call `getUnboundMethod`?

Reply:
IIRC, there were cases where a `MemberReferenceMethod` had a source declaration that was itself a constructed method.